sqlserver/datadog_checks/sqlserver/queries.py

```diff
@@ -214,6 +214,22 @@
     FK.name, FK.parent_object_id, FK.referenced_object_id;
 """
 
+DETECT_DEADLOCK_QUERY = """
+DECLARE @limit INT = ?;
```

Comment:
ideally we should avoid using multi-statements SQL with pyodbc or adodbapi because they don't natively support multi-statements in `execute`. we can try `SELECT TOP ?`

Reply:
oh, ok, let me try, thx.